### PR TITLE
SSCS-8740 - Process A/V Evidence: Send to Judge

### PR DESCRIPTION
**Pull request by ryandac**
SSCS-8740 - Process A/V Evidence: Send to Judge

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java [hunk 1]
```diff
@@ -71,6 +71,10 @@ private void setProcessAudioVideoActionDropdown(SscsCaseData sscsCaseData, boole
             populateListWithItems(listOptions, EXCLUDE_EVIDENCE);
         }
 
+        if (!hasJudgeRole || hasSuperUserRole) {
+            populateListWithItems(listOptions, SEND_TO_JUDGE);
```
Comment (rasikasri) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java:75]: Shouldn't TCW be able to trigger sendToJudge?

**Review by jackmaloney — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java (hunk 1 above)
Comment (ryandac) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java:75]: yes, hence they are not judges

Comment (jackmaloney) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java:75]: Do you even need the 'hasSuperUserRole' part @ryandac ? Isn't it implied that if you are not a judge, then you will always have this.

Comment (ryandac) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java:75]: superusers need to have all options. I wasn't sure if a superuser would also have a judge role, which is why I explicitly add the condition

Comment (jackmaloney) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java:75]: ok fair enough. In reality they should never have both but i guess better to be safe.

Comment (ryandac) [reply src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processaudiovideo/ProcessAudioVideoEvidenceAboutToStartHandler.java:75]: I didn't know, good to know that is the case.

**Pull request merged**